erpnext/selling/sales_common.js

```diff
@@ -486,21 +487,25 @@ erpnext.selling.SellingController = erpnext.TransactionController.extend({
 		}
 
 		this.set_and_update_excise_tax();
+		this.set_and_update_cultivation_tax();
 	},
 
 	item_code: function(doc, cdt, cdn) {
 		this._super(doc, cdt, cdn);
 		if (this.frm.doc.total) {
 			this.set_and_update_excise_tax();
+			this.set_and_update_cultivation_tax();
 		}
 	},
 
 	rate: function(doc, cdt, cdn) {
 		this.set_and_update_excise_tax();
+		this.set_and_update_cultivation_tax();
 	},
 
 	items_remove: function(doc, cdt, cdn) {
 		this.set_and_update_excise_tax();
+		this.set_and_update_cultivation_tax();
```

Comment:
We should include these function calls in `calculate_taxes_and_totals`, these are just too many calls to do individually